tickets/PREOPS-5194

christinawilliams opened the pull request.

christinawilliams committed:
Update portal-dp0-3-5.rst

Updating narrative

christinawilliams committed:
Update portal-dp0-3-5.rst

updating narrative

christinawilliams committed:
Update portal-dp0-3-5.rst

deleted not-applicable new narrative

christinawilliams committed:
Update portal-dp0-3-5.rst

update narrative step 3

christinawilliams committed:
Update portal-dp0-3-5.rst

correction, was for step 2 not 3

christinawilliams committed:
Update portal-dp0-3-5.rst

updates to Step 3.1 and step 3.6

christinawilliams committed:
Update portal-dp0-3-5.rst

more explanation added to step 3.6

christinawilliams committed:
Update portal-dp0-3-5.rst

fixing the additional narrative

christinawilliams committed:
Update portal-dp0-3-5.rst

christinawilliams committed:
Update portal-dp0-3-5.rst

MelissaGraham commented on tutorials-dp0-3/portal-dp0-3-5.rst:
```diff
@@ -114,6 +120,9 @@ Click the "Search" button. This search will return whether any moving object was
 ===============================================================
+This section demonstrates how to upload a user-supplied table and join it with a DP0.3 table.
+
+In this scenario, a list of identifiers (ssObjectId) for moving objects has been assembled by the user and stored in a file (one column, two rows of data, i.e. two independent objects).
```
``ssObjectId`` for correct formatting

MelissaGraham commented on tutorials-dp0-3/portal-dp0-3-5.rst:
```diff
@@ -211,14 +223,13 @@ The query will return all SSSource observation entries for the list of 110 SSObj
     :alt: A screenshot of the fully populated "Object ID Search" section of the UI.
 **A screenshot of the fully populated "Object ID Search" section of the UI.**
```
In all cases for all figures, the caption should be indented (see the rst file for portal tutorial 01) so that it properly appears as a caption associated with the figure.

christinawilliams replied:
Added indentations to the source (but rendered output looks the same to me; so not sure this is what you referred to. Please let me a different edit is needed).

MelissaGraham reviewed (changes requested):
Left a few minor comments, one of which is to fix all figure captions (needs to be indented, see rst file for dp03 portal tutorial 1 for example).

The rendered version is not yet available despite the green check coming up, so can't approve yet until https://dp0-3.lsst.io/v/PREOPS-5194 can be reviewed. Re-request review once you've confirmed the compilation is working?

christinawilliams committed:
Update portal-dp0-3-5.rst

all occurances of SSObjectId -> ssObjectId

christinawilliams committed:
Update portal-dp0-3-5.rst

I (think) this fixes the caption indentation issue

christinawilliams committed:
Update portal-dp0-3-5.rst

added fig numbers

christinawilliams committed:
Update portal-dp0-3-5.rst

removed bold face from fig captions

christinawilliams committed:
Update portal-dp0-3-4.rst

remove bold face from fig caption for portal tutorial 04

christinawilliams committed:
Update portal-dp0-3-3.rst

fixing figure captions for portal tutorial 3

christinawilliams committed:
Update portal-dp0-3-3.rst

fix missed caption and update fig numbering

christinawilliams committed:
Update portal-dp0-3-3.rst

test to get rid of gray bar

christinawilliams committed:
Update portal-dp0-3-3.rst

remove gray bar

christinawilliams committed:
Update portal-dp0-3-2.rst

updating fig caption format to portal tutorial 2

christinawilliams committed:
Update portal-dp0-3-1.rst

update fig caption format to portal tutorial 1

christinawilliams committed:
Update portal-dp0-3-5.rst

removing white trailing spaces

MelissaGraham commented:
As of today (Wed Jun 26), the latest commits (Wed Jun 24) aren't reflected in the rendered version at dp0-3.lsst.io/v, making it not possible to review this branch or proceed. Will need assistance.

christinawilliams committed:
Update table-access-and-queries.rst

fix first warning from github to table-access-and-queries.rst

christinawilliams committed:
Update portal-dp0-3-4.rst

fix github warning to portal-dp0-3-4.rst referring to intro in portal tutorial 3

christinawilliams committed:
Update portal-dp0-3-5.rst

fix incorrect step02a->b figure name in tutorial 5
